ICDO3/load_stage.sql

```diff
@@ -61,6 +61,9 @@ CREATE UNLOGGED TABLE snomed_ancestor AS (
 					order by c.effectivetime desc
 				) as active
 		from sources.sct2_concept_full_merged c
+		-- filter out new sources, as SNOMED update could have been delayed
+		where
+			to_date(c.effectivetime :: varchar, 'yyyymmdd') <= (select to_date(substring(vocabulary_version from 78 for 10),'yyyy-mm-dd') from vocabulary where vocabulary_id = 'SNOMED')
```

Comment:
this subquery 
select to_date(substring(vocabulary_version from 78 for 10),'yyyy-mm-dd') from vocabulary where vocabulary_id = 'SNOMED'
returns 
0001-01-01
Looks wrong